### ISSUE TITLE
[MU4 Issue] Alt+[Shortcut]  then mousing menu has stuck highlighting at times

### ISSUE THREAD

**Issue by worldwideweary** [musescore/MuseScore#10764]
**Describe the bug** 
When accessing above menus via Alt+ whatever like Alt+F for file, then using mouse, the highlighter gets stuck at times.
Not only this, but Left/Right arrows act like Up/Down arrows. 

Expected behavior: Like most decent functioning applications

Initial portion of the following sloppy screencast shows that regular Mouse Activation + Motion seems decent, but using Alt+ then mouse afterwards has the issue:

https://user-images.githubusercontent.com/7139517/157342145-0bb82d4c-1632-463d-ab66-66848c1629f7.mp4

[Linux]

**Comment by Tantacrul** [comment]
@DmitryArefiev - are you able to reproduce this? I think it's possible it might no longer be an issue.

**Comment by DmitryArefiev** [comment]
@Tantacrul @worldwideweary I think that's expected because when using keyboard shortcut (Alt+F) to enable File menu it turns navigation mode and highlighted type is different, it shows highlighted in frame to demonstrate from what element user wants to start navigation. 
But I found related bug when frame of selection remains for top menu. You can see it on video:


https://user-images.githubusercontent.com/90187801/171644617-4f003825-0d6c-4ea2-b54c-e457be93f447.mp4

We can leave the current issue to track that bug

**Comment by DmitryArefiev** [comment]
FYI @Eism

**Comment by Tantacrul** [comment]
> @Tantacrul @worldwideweary I think that's expected because when using keyboard shortcut (Alt+F) to enable File menu it turns navigation mode and highlighted type is different, it shows highlighted in frame to demonstrate from what element user wants to start navigation. But I found related bug when frame of selection remains for top menu. You can see it on video:
> 
>  bandicam.2022-06-02.16-40-01-135.mp4 
> We can leave the current issue to track that bug

I'm wondering if that is even a bug? I mean... it makes sense that highlight would still work for rollover while navigating?

If we agree it is, I'll rewrite the description and delete a few comments to keep it simple.

**Comment by DmitryArefiev** [comment]
> > @Tantacrul @worldwideweary I think that's expected because when using keyboard shortcut (Alt+F) to enable File menu it turns navigation mode and highlighted type is different, it shows highlighted in frame to demonstrate from what element user wants to start navigation. But I found related bug when frame of selection remains for top menu. You can see it on video:
> > bandicam.2022-06-02.16-40-01-135.mp4
> > We can leave the current issue to track that bug
> 
> I'm wondering if that is even a bug? I mean... it makes sense that highlight would still work for rollover while navigating?
> 
> If we agree it is, I'll rewrite the description and delete a few comments to keep it simple.

Yes, the initial bag is fixed. Now the **expected result/actual result is after triggering Alt+F it should show framed highlight** at the top element:

https://user-images.githubusercontent.com/90187801/171851064-8908ec68-1639-4039-9995-7c7b4aad7cb4.mp4

The **BUG** is (as I see for now) - **after triggering Alt+F and navigating to other tabs with mouse or clicking on score, the frame on File is not removed**


https://user-images.githubusercontent.com/90187801/171851700-fc8eee84-84ea-4328-aef4-bd53edc0fa3d.mp4

**Comment by Tantacrul** [comment]
OK, I've changed it to a P3 now and removed all the other comments so it is clear what the issue is.
